packages/E2ETest/windows/ReactUWPTestApp/Package.appxmanifest

```diff
@@ -8,14 +8,14 @@
 
   <Identity
     Name="ReactUWPTestApp"
-    Publisher="CN=RNWTest"
+    Publisher="CN=react-native-windows-testing"
```

Comment:
does this rename work without regenerating the certificates?

Reply:
@asklar: `yarn windows` works fine for playground and e2etests.. 
Is there anything else you'd like me to verify?